espnet2/tasks/abs_task.py

```diff
@@ -1275,6 +1292,8 @@ def main_worker(cls, args: argparse.Namespace):
                 distributed_option=distributed_option,
             )
 
+            wandb.finish() if wandb.run else None
```

Comment:
Sorry, could you explain your intention of this line?  I'm not so familiar with wandb.

Reply:
`wandb.finish()` marks the run as finished.

Reply:
Okay, please change it to if-block.

```python
if wandb.run:
    wandb.finish()
```

Reply:
okay